tools/genevergreen/generate/generate.go

```diff
@@ -229,35 +229,3 @@ func PublishSnapshotTasks(c *shrub.Configuration, toolName string) {
 		false,
 	)
 }
-
-func LocalDeploymentTasks(c *shrub.Configuration, toolName string) {
```

Comment:
took the liberty to remove auto generated tests that were not running any way and had no value to add